src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberContainerSymbol_ImplementationChecks.cs

```diff
@@ -1029,6 +1040,32 @@ bool isOrContainsErrorType(TypeSymbol typeSymbol)
             {
                 return (object)typeSymbol.VisitType((currentTypeSymbol, unused1, unused2) => currentTypeSymbol.IsErrorType(), (object)null) != null;
             }
+
+            static bool isValidNullableOverride(
+                CSharpCompilation compilation,
+                RefKind refKind,
+                TypeWithAnnotations sourceType,
+                TypeWithAnnotations targetType)
+            {
+                var conversions = compilation.Conversions.WithNullability(true);
+                switch (refKind)
+                {
+                    case RefKind.Ref:
+                        // ref variables are invariant
+                        return sourceType.Equals(
+                            targetType,
+                            TypeCompareKind.AllIgnoreOptions & ~(TypeCompareKind.IgnoreNullableModifiersForReferenceTypes));
+
+                    case RefKind.Out:
+                        // out variables have inverted variance
+                        (sourceType, targetType) = (targetType, sourceType);
+                        break;
+
+                    default:
+                        break;
+                }
+                return conversions.HasAnyNullabilityImplicitConversion(sourceType, targetType);
```

Comment:
>return conversions.HasAnyNullabilityImplicitConversion(sourceType, targetType); [](start = 16, length = 79)

It doesn't look like this PR covers scenarios with implicit/explicit interface implementations. Is the plan to cover them in a separate PR?

Reply:
Yup, this is just overriding.